src/test/java/org/example/AthleteScoring.java

```diff
@@ -0,0 +1,35 @@
+package org.example;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Collection;
+
+import static org.junit.Assert.assertEquals;
+
+public class AthleteScoring {
+
+    AthleteScoring testAthleteScoring = new AthleteScoring ();
+
+    @Before
+    public void setUp(){
+        testAthleteScoring.readAthletes();
+    }
+
+    private void readAthletes() {
+    }
+
+
+    @Test
+    public void readAthletesTest() {
+
+        assertEquals(testAthleteScoring.getAthletes().size(), 3);
```

Comment:
revers order, expected, actual result. so 3, myMethod. Also, does this test work? you haven't added any athletes here. Don't test reading and parsing csv files all in one test, create a separate one just for parsing the data